src/game_renderer/GameRenderer.cpp

```diff
@@ -39,6 +50,24 @@ void GameRenderer::Init(const std::string& resource_dir) {
   color_vec.push_back(glm::vec3(89.0/255.0, 236.0/255, 0));
   color_vec.push_back(glm::vec3(0/255.0, 172.0/255, 236.0/255.0));
   color_vec.push_back(glm::vec3(150.0/255.0, 0/255, 236.0/255.0));
+    
+  texture0 = std::make_shared<Texture>();
+  texture0->setFilename(std::string(ASSET_DIR) + "/textures/" + "rainbow.jpg");
```

Comment:
textureRainbow or something similar might be a better name
@alexottoboni would it be better/reasonable to load the texture info from json?